.storybook/manager-head.html

```diff
@@ -68,18 +68,34 @@
 		margin-left: 0.5rem !important;
 	}
 
+	.sidebar-item > a {
+		display: flex;
+		align-items: center;
+		gap: 0.25rem;
+	}
+
+	.sidebar-item[data-nodetype='document'] > a {
+		padding-left: 1em !important;
+		gap: calc(0.25rem + 0.2em) !important;
+	}
+
+	.sidebar-item[aria-expanded='true'] + .sidebar-item[data-nodetype='document'] > a {
+		padding-left: 2.6em !important;
+		gap: calc(0.25rem + 0.2em) !important;
+	}
+
 	.sidebar-item,
 	.sidebar-subheading {
 		margin-bottom: 1px !important;
 	}
 
-	.sidebar-subheading ~ .sidebar-item {
+	.sidebar-subheading~.sidebar-item {
```

Comment:
Why the squeeze? Not applied above 🤔
Can it be the same across the file?

Reply:
Damnit, auto-formatting 😅 
I'll fix it